tools/smina/smina.xml

```diff
@@ -0,0 +1,364 @@
+                <option value="false" selected="true">Do not use this option</option>
+                <option value="true">Use this option</option>
+            </param>
+            <param argument="--print_atom_types" type="select" label="Print all available atom types" >
+                <option value="false" selected="true">Do not use this option</option>
+                <option value="true">Use this option</option>
+            </param>
+        </section>
+        <section name="misc" title="Misc" >
+            <param argument="--cpu" type="integer" label="The number of CPUs to use" optional="true" />
+            <param argument="--seed" type="float" label="Explicit random seed" optional="true" />
+            <param argument="--exhaustiveness" type="integer" value="8" label="Exhaustiveness of the global search" />
+            <param argument="--num_modes" type="integer" value="9" label="Maximum number of binding modes to generate" >
+                <validator type="in_range" min="1"/>
+            </param>
+            <param argument="--energy_range" type="float" value="3" label="Maximum energy difference between the best binding mode and the worst one displayed (kcal/mol)" />
+            <param argument="--min_rmsd_filter" type="float" value="1" label="Rmsd value used to filter final poses to remove redundancy" />
+            <param argument="--addH" type="select" label="Automatically add hydrogens in ligands" >
+                <option value="false" >Do not use this option</option>
+                <option value="true" selected="true">Use this option</option>
+            </param>
+        </section>
+    </inputs>
+    <outputs>
+        <data name="output" format="txt" />
+        <data name="out_flex_output" format="txt" label="smina: flexible receptor residues">
+            <filter>output_sect['out_flex']=="true"</filter>
+        </data>
+        <data name="output_log" format="txt" label="smina: log file">
+            <filter>output_sect['log']=="true"</filter>
+        </data>
+        <data name="atom_terms_output" format="txt" label="smima: per-atom interaction term values">
+            <filter>output_sect['atom_terms']=="true"</filter>
+        </data>
+    </outputs>
+    <tests>
+        <test>
+            <param name="receptor" value="smina/protein.pdbqt" />
+            <param name="ligand" value="smina/ligand.pdbqt" />
+            <param name="center_x" value="11" />
+            <param name="center_y" value="90.5" />
+            <param name="center_z" value="57.5" />
+            <param name="size_x" value="22" />
+            <param name="size_y" value="24" />
+            <param name="size_z" value="28" />
+            <output name="stand_output" >
+                <assert_contents>
+                    <has_text_matching expression="END" />
+                </assert_contents>
+            </output>
```

Comment:
Since you're including 2 output files, I suppose you should change this to `<output ... file="..." />`